### DISCUSSION TITLE
crash & bluescreen

### DISCUSSION THREAD

**Discussion by techioo** [flxzt/rnote#1230] [Q&A]
My computer crashed to a blue screen due to a rnote glitch.  anybody had this problem?  
i was taking note on a PDF file, and it was smoth then i want to adjust to a grid layout and it froze then bluecreen 

- rnote version: rnote-win-installer-0.11.0-x86_64
- computer:  AMD Ryzen 7 6800U with Radeon Graphics            2.70 GHz
- OS: windows
should i offer any info?
i lost all the stuff again.. the first time i used rnote i didnt save, and this time again... 
all my notesss gone, but anyway, i still like rnote, just anybody could help me out?

**Comment by Doublonmousse** [comment]
That's definitely unusual. Do you have up to date windows/graphics drivers? Do you have a blue screen error code?

**Comment by techioo** [comment]
unfortunately, i didnt got the screenshot and the bluescreen time was quite short, didnt have the time to remember..
but i think next time will come soon, i will pay a serious attenttion to it
